src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -1,10 +1,31 @@
 (ns proton.layers.core.keybindings
   (:require [proton.layers.core.actions :as actions :refer [state]]
+            [proton.lib.pane_manager :as panes]
             [proton.lib.proton :as proton]))
 
+(defn select-window-fn [n]
```

Comment:
__:fx__ should be a function, so I wrap panes/focus-on-item into function that returns function + window number as argument.